.github/CODEOWNERS

```diff
@@ -861,6 +863,8 @@ lib/gi/search_client.rb @department-of-veterans-affairs/my-education-benefits @d
 lib/gi/search_configuration.rb @department-of-veterans-affairs/my-education-benefits @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 lib/github_authentication @department-of-veterans-affairs/vsp-identity
 lib/hca @department-of-veterans-affairs/vfs-10-10 @department-of-veterans-affairs/vfs-authenticated-experience-backend @department-of-veterans-affairs/backend-review-group @department-of-veterans-affairs/va-api-engineers
+lib/form1010_ezr @department-of-veterans-affairs/vfs-10-10 @department-of-veterans-affairs/vfs-authenticated-experience-backend @department-of-veterans-affairs/backend-review-group @department-of-veterans-affairs/va-api-engineers
+lib/va_1010_forms @department-of-veterans-affairs/vfs-10-10 @department-of-veterans-affairs/vfs-authenticated-experience-backend @department-of-veterans-affairs/backend-review-group @department-of-veterans-affairs/va-api-engineers
```

Comment:
should delete `@department-of-veterans-affairs/vfs-authenticated-experience-backend`

Reply:
Removed!